agreed/0009-MPID-Messaging/0009-MPID-Messaging.md

```diff
@@ -93,16 +93,11 @@ This can be implemented as a `Vec<(sender_name: ::routing::NameType, sender_publ
 
 ## Messaging Format Among Nodes
 
-Messages between Clients and MpidManagers will utilise [`::routing::structured_data::StructuredData`][5], for example:
+Messages between Clients and MpidManagers will utilise [`::routing::plain_data::PlainData`][5], for example:
 ```rust
-let sd = StructuredData {
-    type_tag: MPID_MESSAGE,
-    identifier: mpid_message_name(mpid_message),
-    data: ::utils::encode(mpid_message),
-    previous_owner_keys: vec![],
-    version: 0,
-    current_owner_keys: vec![sender_public_key],
-    previous_owner_signatures: vec![]
+let pd = PlainData {
+    name: mpid_message_name(mpid_message),
+    data: serialise(MpidMessageWrapper::PutMessage(mpid_message)),
```

Comment:
Probably best to follow https://maidsafe.atlassian.net/browse/MAID-1550 and update it with whatever we need.